test/Tmds.Ssh.Tests/SftpClientTests.cs

```diff
@@ -686,13 +686,19 @@ public async Task UploadDownloadFile(int fileSize)
             // Download
             await sftpClient.DownloadFileAsync(remotePath, destinationPath);
 
+            var downloadStream = new MemoryStream();
```

Comment:
Instead of adding it here, can you add a separate test. Something like:

```cs
[Fact]
DownloadFileToStream()
{
  using var sftpClient = await _sshServer.CreateSftpClientAsync(sftpExtensions);

  (string sourceFileName, byte[] sourceData) = await CreateRemoteFileWithRandomDataAsync(sftpClient, fileSize: 100);

  ...
}
```

Reply:
will do